Tests/ML/test_lightning_containers.py

```diff
@@ -330,19 +330,18 @@ def test_innereyecontainer_setup_passes_on_allow_incomplete_labels(
     config.set_output_to(test_output_dirs.root_dir)
     config.allow_incomplete_labels = allow_partial_ground_truth
     container = InnerEyeContainer(config)
-    test_done_message = "Stop now, the test has passed."
 
     def mocked_convert_channels_to_file_paths(
             _: List[str],
             __: pd.DataFrame,
             ___: Path,
             ____: str,
             allow_incomplete_labels: bool) -> Tuple[List[Optional[Path]], str]:
+        paths: List[Optional[Path]] = []
+        failed_channel_info = ''
         assert allow_incomplete_labels == allow_partial_ground_truth
-        raise RuntimeError(test_done_message)
+        return paths, failed_channel_info
 
-    with pytest.raises(RuntimeError) as runtime_error:
-        with mock.patch("InnerEye.ML.lightning_base.convert_channels_to_file_paths") as convert_channels_to_file_paths_mock:
-            convert_channels_to_file_paths_mock.side_effect = mocked_convert_channels_to_file_paths
-            container.setup()
-    assert str(runtime_error.value) == test_done_message
+    with mock.patch("InnerEye.ML.lightning_base.convert_channels_to_file_paths") as convert_channels_to_file_paths_mock:
+        convert_channels_to_file_paths_mock.side_effect = mocked_convert_channels_to_file_paths
+        container.setup()
```

Comment:
... do you want to assert that the mock was called? And maybe add in comments why it is important to check for that?

Reply:
I don't think I need to assert that the mock's called itself, I just need to know that if the function under test calls the function being mocked, it does it with the correct parameter.

Reply:
I get you, it's about function renaming. I'll fix that.